config/database.yml

```diff
@@ -13,20 +13,20 @@ default: &default
   port: <%= ENV['OSEM_DB_PORT'] || '5432'  %>
   username: <%= ENV['OSEM_DB_USER'] || 'postgres' %>
   password: <%= ENV['OSEM_DB_PASSWORD'] || 'mysecretpassword' %>
-  database: <%= ENV['OSEM_DB_NAME'] || 'postgres' %>
   pool: 5
   timeout: 5000
 
 development:
   <<: *default
-  database: osem_development
+  database: <%= ENV['OSEM_DB_ADAPTER'] == 'sqlite3' ? "db/development.sqlite3" : "osem_development" %>
```

Comment:
Hm can you do this above where we set the encoding too? Something like

```
encoding = 'unicode'
db_prefix = ''
if ENV['OSEM_DB_ADAPTER'] == 'mysql2'
     encoding = 'utf8'
elsif  ENV['OSEM_DB_ADAPTER'] == 'sqlite3'
   db_prefix = 'db/'
end
...
development:
  <<: *default
  database: <%= db_prefix + 'osem_development' %>
```

Reply:
We need the suffix too... although there's no hard requirement, it's conventional to have `.sqlite3` files.